rest_api/packaging/rest_api.toml.example

```diff
@@ -0,0 +1,15 @@
+#
+# Sawtooth Lake -- REST API Configuration
+#
+
+# The port for the api to run on
+#   port = 8080
```

Comment:
The port/host configuration here is significantly different than the *_endpoint configuration options in the validator, but both are what to listen on.  We should probably make them consistent.